lib/liquid/combinators/tags/if.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Liquid.Combinators.Tags.If do
+  @moduledoc """
+  Executes a block of code only if a certain condition is true.
+  If this condition is false executes `else` block of code
+  Input:
+  ```
+  {% if product.title == 'Awesome Shoes' %}
+    These shoes are awesome!
+  {% else %}
+    These shoes are ugly!
+  {% endif %}
+  ```
+  Output:
+  ```
+    These shoes are ugly!
+  ```
+  """
+
+  import NimbleParsec
+  alias Liquid.Combinators.Tag
+
+  def elsif_tag, do: Tag.define("elsif", &predicate/1)
+
+  def else_tag, do: Tag.define("else")
+
+  def unless_tag, do: Tag.define("unless", &predicate/1, &body/1, "endunless")
+
+  def tag, do: Tag.define("if", &predicate/1, &body/1, "endif")
+
+  defp body(combinator) do
+    combinator
+    |> optional(parsec(:__parse__))
+    |> optional(times(parsec(:elsif_tag), min: 1))
+    |> optional(times(parsec(:else_tag), min: 1))
```

Comment:
Is it possible to have multiple `else` tags in a body?

Reply:
Yes, indeed it is a strange behavior in liquid. It is implemented this way in versions 3 and 4 and in liquid-elixir too